files/image_config/hostcfgd/hostcfgd

```diff
@@ -219,6 +220,537 @@ class AaaCfg(object):
+            subprocess.check_call(cmd, shell=True)
+        except subprocess.CalledProcessError as err:
+            syslog.syslog(syslog.LOG_ERR, "{} - failed: return code - {}, output:\n{}".format(err.cmd, err.returncode, err.output))
+            pass
+
+    def mgmt_intf_dhclient_set (self, ifname, attr, enable):
+        version  = ""
+        file_ext = ""
+        cmd_opt  = ""
+
+        if attr == "ipv6_dhcp_client":
+            version = "-6"
+            file_ext = "6"
+            cmd_opt = "-D LL"
+
+        if enable == True:
+            try:
+                path = "/var/run/dhclient{}.{}.pid".format(file_ext, ifname)
+                if os.path.exists(path):
+                    cmd = "[ -f /var/run/dhclient{}.{}.pid ] && kill `cat /var/run/dhclient{}.{}.pid` && rm -f /var/run/dhclient{}.{}.pid".format(file_ext, ifname, file_ext, ifname, file_ext, ifname)
+                    subprocess.check_call(cmd, shell=True)
+            except subprocess.CalledProcessError as err:
+                pass
+
+        try:
+            if enable == True:
+                cmd = "/sbin/dhclient {} -pf /run/dhclient{}.{}.pid -lf /var/lib/dhcp/dhclient{}.{}.leases {} -nw {} ".format(version, file_ext, ifname, file_ext, ifname, ifname, cmd_opt)
+            else:
+                cmd = "/sbin/dhclient {} -r {} && [ -f /var/run/dhclient{}.{}.pid ] && kill `cat /var/run/dhclient{}.{}.pid` && rm -f /var/run/dhclient{}.{}.pid".format(version, ifname, file_ext, ifname, file_ext, ifname, file_ext, ifname)
+
+            syslog.syslog(syslog.LOG_INFO, "Configure dhclient , cmd : {}".format(cmd))
+
+            subprocess.check_call(cmd, shell=True)
+        except subprocess.CalledProcessError as err:
+            syslog.syslog(syslog.LOG_ERR, "{} - failed: return code - {}, output:\n{}".format(err.cmd, err.returncode, err.output))
+            pass
+
+    def mgmt_intf_ip_prefix_set (self, ifname, ip_prefix, op):
+        try:
+            ver = "-4"
+            mgmt_conf = netaddr.IPNetwork(ip_prefix)
+            if mgmt_conf.version == 6:
+                ver = "-6"
+            cmd = "ip {} addr {} {} dev {}".format(ver, op.lower(), ip_prefix, ifname)
+            syslog.syslog(syslog.LOG_INFO, "Configure ip_prefix, op : {}, ip_prefix: {}, cmd : {}, mgmt_conf: {}".format(op, ip_prefix, cmd, mgmt_conf))
+            subprocess.check_call(cmd, shell=True)
+            cmd = "ip {} rule {} from {} table default pref 1003".format(ver, op.lower(), str(mgmt_conf.ip))
+            subprocess.check_call(cmd, shell=True)
+        except subprocess.CalledProcessError as err:
+            syslog.syslog(syslog.LOG_ERR, "{} - failed: return code - {}, output:\n{}".format(err.cmd, err.returncode, err.output))
+            pass
+
+    def handle_mgmt_port_cfg(self, key, data):
+        mgmt_intf =  self.app_db.get_entry(APP_MGMT_PORT_TABLE, key)
+        is_all = False
+        if mgmt_intf == {}:
+            is_all = True
```

Comment:
can you make it verbose? What is intended here with `is_all`

Reply:
Modified the variable name to cfgAllAtrr .